src/foundry/client/pixi/layers/controls/layer.d.ts

```diff
@@ -169,18 +174,20 @@ declare global {
      */
     drawOffscreenPing(
       position: PIXI.Point,
-      options?: PingOptions & {
-        /**
-         * The style of ping to draw, from CONFIG.Canvas.pings.
-         * @defaultValue `"arrow"`
-         */
-        style?: string;
-
-        /**
-         * The user who pinged.
-         */
-        user?: InstanceType<ConfiguredDocumentClass<typeof User>>;
-      },
+      options?: InexactPartial<
+        PingOptions & {
```

Comment:
Are the `PingOptions` all optional? If not change this to be `PingOptions & InexactPartial<...>`

Reply:
Yes, the PingOptions are optional